### PR TITLE
Flexran: Initial commit

### PR DESCRIPTION
**Pull request by HughNhan**
Issue: This is the initial Flexran commit.

Description:

1. run.sh  -  'bash run.sh' to run flexran workload. 
2. annotations.json  - For low latency
4. mv-params.json  - This file produces 3 iteration: 'runall 0 0 5', 'run 2 0 5 1001', and run a suite specified by '/opt/flexran/tests/nr5g/fd/testmac_fd_mu0_5mhz.cfg
 5. resources.json -  CPU, mem and FEC h/w required resources
6. securityContext.json
7. tool-params.json
8. volumeMounts.json - For DPDK and devel flexran hostPath mount
9. volumes.json - For DPDK and flexran hostPath mount

Prerequisite:

1. At the moment, the container image does not contains Flexran S/W since it is big, ~100GB (bytes!).  Instead, we copy flexran S/W to worker node(s) and make the k8s endpoints's container mount it.  
2. To run with HW FEC the worker node must have H/W FEC i.e N3000 installed and properly configured i.e flash and ONEness SRIOV operator.

### CODE REVIEW

**Review by k-rister — COMMENTED** [review]
checking if I can comment here...its acting weird when I try to review the patch...

**Comment by k-rister** [comment]
> checking if I can comment here...its acting weird when I try to review the patch...

so this part works at least...

**Comment by HughNhan** [comment]
> checking if I can comment here...its acting weird when I try to review the patch...
Probably something transient. Could you try again.

File: flexran/run.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# ocp admin node
+k8shost1=perf176a.perf.lab.eng.bos.redhat.com
```
Comment (k-rister) [inline flexran/run.sh:4]: To be consistent with other workloads (trafficgen) lets obfuscate these hostnames:

```
$ git grep redhat
trafficgen/run.sh:remhost1=perfxxx.perf.lab.eng.bos.redhat.com
trafficgen/run.sh:k8shost1=perfxxx.perf.lab.eng.bos.redhat.com
```

We probably should not even include anything under redhat.com anywhere...

File: flexran/securityContext.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+"securityContext": {
+  "privileged": true,
```
Comment (k-rister) [inline flexran/securityContext.json:2]: Are we sure we need this?  Or are we working on figuring out how to not need this?

File: flexran/run.sh (hunk 1 above)
Comment (HughNhan) [reply flexran/run.sh:4]: OK. I will make this change.

File: flexran/securityContext.json (hunk 2 above)
Comment (HughNhan) [reply flexran/securityContext.json:2]: Yes. We need it for mount hostPath.

**Review by k-rister — APPROVED** [review]

**Pull request merged**